client/src/reducers/lessonReducer.ts

```diff
@@ -25,6 +26,8 @@ export interface Lesson {
 
 export type LessonState = {[id: number]: Lesson};
 
+export const parts_of_speech = ['noun', 'verb', 'adjective', 'adverb'];
```

Comment:
He mentioned in the meeting that he only uses those 4, that's why I removed the rest. Do we still want all of them anyways even if most won't ever be used?